src/schema/v2/show.ts

```diff
@@ -112,8 +112,69 @@ export const ShowType = new GraphQLObjectType<any, ResolverContext>({
           return artists
         },
       },
+      allArtworksConnection: {
```

Comment:
Can this be simply `artworksConnection`?

Reply:
there is already an `artworksConnection` but it only returns unpublished artworks as far as I can tell so I named this allArtworksConnection to distinguish the 2

Reply:
Perhaps we could [update the arguments](https://github.com/artsy/metaphysics/blob/339aca96e75135a42e7e137ae7f0bfc308ad75fa/src/schema/v2/show.ts#L87), or adjust the loader so there's no need for duplication

Reply:
I think we can and this extra field is totally unnecessary, thanks!